persistence/src/main/scala/hmda/persistence/processing/SubmissionManager.scala

```diff
@@ -107,9 +107,9 @@ class SubmissionManager(submissionId: SubmissionId) extends HmdaActor {
       log.info(s"Validation completed with errors for submission: ${sId.toString}")
       submissionFSM ! CompleteValidationWithErrors
 
-    case Signed(sId) =>
-      log.info(s"Submission signed: ${sId.toString}")
-      submissionFSM ! Sign
+    case Signed =>
+      log.info(s"Submission signed: ${submissionId.toString}")
+      submissionFSM forward Sign
```

Comment:
For the endpoint to work as designed, it needs a response from the actor that says whether the `Sign` action was successful (so then we can know whether to give a `202` or `400` response). 

So, the API code sends the `Sign` event, the `SubmissionManager` forwards it to the FSM, then (because it was `forward`), the FSM responds back to the API code. The FSM responds `Some(Signed)` if the state transition was successful, or `None` if it wasn't.